acceptance_tests/cypress/integration/lotse_flow.spec.js

```diff
@@ -299,7 +299,7 @@ context('Acceptance tests', () => {
                 cy.get('#person_a_town').type(taxReturnData.personA.town)
                 cy.get(submitBtnSelector).click()
```

Comment:
Also general: I think we need to adapt the Mock Erica to reflect the changes in Erica

Reply:
Thank you, I forgot to change `is_person_a_account_holder` in line 21 of MockErica. Do you see anything else besides changing the required fields?